app/controllers/admin/booths_controller.rb

```diff
@@ -38,6 +38,24 @@ def create
       @booth.submitter = current_user
 
       if @booth.save
+
+        if booth_params[:invited_users]
+          emails_array = booth_params[:invited_users].split(',')
+
+          emails_array.each do |email|
+            if User.find_by(email: email).nil? || ((!User.find_by(email: email).confirmed? &&
```

Comment:
why need the email to be confirmed?

Reply:
why are you looking by email if you also have the id in the view?

Reply:
how can it happen that the user doesn't exist? shouldn't the user get an error in this case?

Reply:
> why need the email to be confirmed?

I want that only unconfirmed users can receive a new invitation so I used negation before that.
 

> why are you looking by email if you also have the id in the view?

@Ana06 Which id? I am confused.


> how can it happen that the user doesn't exist? shouldn't the user get an error in this case?

@Ana06  What if email is not valid (say: abc) so in that case we must add a condition to check it and if an email id is not correct then i have added a javascript check for email id (this condition will prevent error if user has disabled javascript). So I found no reason for sending an error to user since he cannot add any string other than email.

Reply:
@rahul2240 

> I want that only unconfirmed users can receive a new invitation so I used negation before that.

I am not sure I understand you. Can you elaborate?

> @Ana06 Which id? I am confused.

User id

> What if email is not valid (say: abc) so in that case we must add a condition to check it and if an email id is not correct then i have added a javascript check for email id (this condition will prevent error if user has disabled javascript). So I found no reason for sending an error to user since he cannot add any string other than email.

you can always modify the Javascript from the user side. But I would say before entering on those details, we should focus on the workflow. Please check https://github.com/openSUSE/osem/pull/2535#discussion_r292991717

Reply:
> I am not sure I understand you. Can you elaborate?
> 
> >

@Ana06 if a user is confirmed then we don't want to send an invitation for registration in OSEM. 
But if the user is unconfirmed, then we should send an invitation again to him to join the app and become booth responsible.

> User id

@Ana06  we have user id in view only for those who are currently responsibles, but here, we are asking the creators to enter email id of the users.

Reply:
> @Ana06 if a user is confirmed then we don't want to send an invitation for registration in OSEM.
But if the user is unconfirmed, then we should send an invitation again to him to join the app and become booth responsible.

a user could have created the account himself but not being confirmed, right?

Reply:
@Ana06 I have updated the condition after having a discussion on it. Please see https://github.com/openSUSE/osem/pull/2535#discussion_r295793443

Now only those users will be invited who haven't logged in to OSEM before.